images/miq-app/docker-assets/appliance-initialize.sh

```diff
@@ -34,6 +34,9 @@ case $? in
 
     # Restore symlinks from PV to application rootdir
     restore_pv_data
+
+    # Set admin pwd
```

Comment:
This comment hardly seems necessary.

Reply:
@chessbyte Surely , already commented and working on requested changes..